client/src/components/common/Header/index.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { MenuIcon, headerLogo, MenuLogo } from '../../assets';
+import menuData from './menuData';
+import Button from '../Button';
+import './style.css';
+
+class Header extends Component {
+  state = {
+    isClosed: true,
+  };
+
+  onclick = () => {
+    const { isClosed } = this.state;
+    this.setState({ isClosed: !isClosed });
+  };
+
+  onclickForModel = e => {
+    if (e.target.id === 'model') {
+      this.setState({ isClosed: true });
+    }
+  };
+
+  render() {
+    const { isClosed } = this.state;
+
+    return (
+      <div className="header">
+        <div className="header__bar">
+          <Button className="menu-icon" onClick={this.onclick}>
+            <MenuIcon />
+          </Button>
+          <Link to="home" className="header-logo">
+            <img src={headerLogo} alt="header logo" />
+          </Link>
+        </div>
+        <div
+          className={!isClosed ? 'menu__model' : ''}
+          onClick={this.onclickForModel}
+          id="model"
+        >
+          <nav className={isClosed ? 'hide' : 'menu__nav'}>
+            <div className="menu__head" onClick={this.onclick}>
```

Comment:
This will resolve eslint error mostly
```suggestion
            <div  className="menu__head"  role="button" tabIndex={0} onClick={this.onclick}>
```

Reply:
Done